internal/pkg/service/stream/plugin/collection.go

```diff
@@ -37,9 +37,45 @@ func (c *Collection) OnBranchSave(fn onBranchSaveFn) {
 	c.onBranchSave = append(c.onBranchSave, fn)
 }
 
+func (c *Collection) OnBranchEnable(fn onBranchSaveFn) {
+	c.onBranchSave = append(c.onBranchSave, func(ctx context.Context, now time.Time, by definition.By, original, branch *definition.Branch) error {
```

Comment:
It is `original` and `updated` version of the `branch`.

If I use `updated` instead of `branch`, it leads to errors that it is not easy to see in the code below what the object is.
`original` value is used rarely.

Reply:
https://github.com/keboola/keboola-as-code/pull/1792/commits/746f67df64188dc4de776baa4858614f378da6c1

Reply:
@jachym-tousek-keboola @hosekpeter @Matovidlo 

Please review actual naming in https://github.com/keboola/keboola-as-code/blob/michaljurecko-PSGO-596-atomic-op-store/internal/pkg/service/stream/plugin/collection.go

It is now clear, or do you have some suggestions?

Reply:
In my opinion it looks consistent and fine. No adjustements needed for me, variable name presents the type. :+1: